### DISCUSSION TITLE
Update a CustomField Value

### DISCUSSION THREAD

**Discussion by crouser7** [zapadi/redmine-net-api#348] [General]
When trying to update the Value of a Custom Field i get the following Errors:
for Using "Issue.CustomFields[i].Value = 1;" the property is readonly
for Using "Issue.CustomFields[i].Values[0] = 1;" i get int cannot implicitly converted to "Redmine.Net.Api-Types.CustomFieldValue"

how can i edit CustomField Values?

**Comment by zapadi** [comment]
Hi,

In order to update an issue custom field, you have to do it like this:
```
issue.CustomFields[i].Values[0].Info = 1.ToString(CultureInfo.InvariantCulture);
```

**Reply by crouser7** [reply]
Awesome that worked, thanks alot.
